de.metas.contracts/src/main/java/de/metas/flatrate/api/IFlatrateHandler.java

```diff
@@ -49,4 +49,20 @@
 	 * @task https://github.com/metasfresh/metasfresh/issues/549
 	 */
 	void afterExtendFlatrateTermCreated(final I_C_Flatrate_Term oldTerm, final I_C_Flatrate_Term newTerm);
+	
+	
+	/**
+	 * Invoked by {@link IFlatrateBL#de.metas.flatrate.api.impl.FlatrateBL.extendContract0(I_C_Flatrate_Term, boolean, boolean, I_C_OrderLine, String)} after the given <code>term</code> was checked.
```

Comment:
Checked refers to the fact that the flatrate term is not autorenew. So was checked for that.
Ended form the method, is referring to the fact that the contract has expired, and if it does need to be extended, then the method is applying some code, after the contract has ended,